source/containers.tex

```diff
@@ -700,6 +700,31 @@
 
 \end{libreqtab4a}
 
+\pnum
+In this Clause the behavior of certain member functions and deduction guides
```

Comment:
While we are in the vicinity, `basic_string`'s [deduction guide specification](https://github.com/cplusplus/draft/blob/master/source/strings.tex#L1491) is missing a `\pnum`.